.teamcity/OpenSourceProjects_Storybook/buildTypes/OpenSourceProjects_Storybook_CliTest.kt

```diff
@@ -27,7 +27,7 @@ object OpenSourceProjects_Storybook_CliTest : BuildType({
         }
         script {
             name = "Test"
-            scriptContent = "yarn test --cli"
+            scriptContent = "./scripts/run-e2e.sh"
```

Comment:
Looks like `yarn test --cli` won't work without local registry anyway, so maybe it should start it always?

Reply:
yeah that's a fair point.